Community/Tdarr_Plugin_x7ac_Remove_Closed_Captions.js

```diff
@@ -1,62 +1,54 @@
-/* eslint-disable */
 function details() {
   return {
-    id: "Tdarr_Plugin_x7ac_Remove_Closed_Captions",
-    Stage: "Pre-processing",
-    Name: "Remove closed captions",
-    Type: "Video",
-    Operation: "Remux",
+    id: 'Tdarr_Plugin_WS_Remove_Closed_Captions',
+    Stage: 'Pre-processing',
+    Name: 'Remove burned closed captions',
+    Type: 'Video',
+    Operation: 'Remux',
     Description:
-      "[Contains built-in filter] If detected, closed captions (XDS,608,708) will be removed.",
+      '[Contains built-in filter] If detected, closed captions (XDS,608,708) will be removed. Derived from x7ac Closed Captions plugin.',
     Version: "1.00",
     Link:
-      "https://github.com/HaveAGitGat/Tdarr_Plugins/blob/master/Community/Tdarr_Plugin_x7ac_Remove_Closed_Captions.js",
-    Tags: "pre-processing,ffmpeg,subtitle only",
+      'https://github.com/HaveAGitGat/Tdarr_Plugins/blob/master/Community/',
```

Comment:
Just removing the file from the link doesn't solve the length being too long, cut it up to two strings and concatenate them.

Reply:
My bad. I forgot to adjust that one. I shouldn't have touched it.